concepts/strings/introduction.md

```diff
@@ -53,3 +53,13 @@ new_release.find("app");
 new_release.find("!");
 // => 24
 ```
+
+There is also the `to_string` function, which can be used to convert integers and float/double values to string.
```

Comment:
Nice one. I would  change `to_string` to `std::to_string` to underline how it is used. Same in the next line.

Reply:
> They will then populate the `std` namespace.

Yeah, thought of it, then just saw this line. So, i assumed it is already said.